xc7/primitives/bram/bram.pb_type.xml

```diff
@@ -311,7 +311,11 @@ reason, BRAM tiles use a "fused sites" mode to accomidate the cross site logic.
      <pb_type name="RAMB18E1_Y0_IN" num_pb="1" blif_model=".subckt RAMB18E1_VPR">
       <xi:include href="ramb18e1.pb_type.xml" xpointer="xpointer(pb_type/child::node()[local-name()!='metadata'])" />
       <metadata>
-       <xi:include href="ramb18e1.pb_type.xml" xpointer="xpointer(pb_type/metadata/child::node())" />
+       <meta name="fasm_params">
+        <xi:include href="ramb18e1.pb_type.xml" xpointer="xpointer(pb_type/metadata/meta/child::node())" />
```

Comment:
This xpointer:
```
xpointer="xpointer(pb_type/metadata/meta/child::node())"
```
is not specific enough.  The removed include from on line ~314 should read "include all metadata children that are **not** `name="fasm_params"`, and the new include from line 315 should read "include meta child where attribute `name="fasm_params"`.

Reply:
If xpointer cannot be that specific, this is an ok solution.

Reply:
I am trying to see if looking up to attributes work, in case I have moved the other metadata out of ramb18e1.pb_type, so that the only meta there is related to fasm_params.

Reply:
It's just a nasty surprise to leave behind.  I'm pretty sure this can be done, here is a link to a similiar example: https://stackoverflow.com/questions/10628431/html-xpath-selectively-avoiding-tags-when-extracting-text

Combined with https://www.w3.org/TR/1999/REC-xpath-19991116/#attribute-nodes

Reply:
Done, now fasm_params is extracted from ramb18e1 pb_type and the other meta are just being copied.